Maya Protocol/chain.json

```diff
@@ -0,0 +1,74 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mayachain",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mayaprotocol.com/",
+  "pretty_name": "Maya Protocol",
+  "chain_id": "mayachain-mainnet-v1",
+  "bech32_prefix": "maya",
+  "daemon_name": "mayanode",
+  "node_home": "$HOME/.mayanode",
+  "slip44": 931,
+  "codebase": {
+    "git_repo": "https://gitlab.com/mayachain/mayanode",
+    "recommended_version": "mainnet",
```

Comment:
it is unheard of to have version names that don't have numbers. This area is not meant to clarify mainnet vs testnet. I'd expect something like 'v1.0'...

Reply:
Updated everything.